pvl/__init__.py

```diff
@@ -60,20 +62,27 @@ def load(path, parser=None, grammar=None, decoder=None, **kwargs):
 
 def get_text_from(path) -> str:
     try:
-        p = Path(path)
-        return p.read_text()
+        if isinstance(path, str) and path.startswith('http'):
```

Comment:
Since `urllib.request.urlopen()` takes a string or a Request object, we might as well allow Request objects here, too.  String-wise, is 'http' enough?  We should probably use urllib.parse to verify the string qualifies as a URL.  Surely, urlopen does that, but we need to distinguish one of these strings instead of one of those, so we can branch appropriately.  Unless we try and urlopen first on *any* string, and if it Errors, try parsing it as regular PVL-text.  If-statements vs try-except, potato - potahto.

Reply:
yeah there is probably a base class in the std lib to check against, and yeah verifying urls is a good idea.. I personally prefer if statements to try/catches as one could imagine multiple different underlying methods throwing the same exception type.. otherwise it sort of feels like trying to put in a usb thumb-drive repeatedly until it works